pkg/jx/cmd/import.go

```diff
@@ -488,14 +488,14 @@ func (options *ImportOptions) DraftCreate() error {
 		return err
 	}
 
-	provider, err := gits.CreateProvider(options.GitServer, options.GitUserAuth, options.Git())
```

Comment:
1. options.GitProvider is already created before calling DraftCreate(). No need to create provider here.
2. In case you run import via 'jx import --org orgname  --github' and call the import in 2nd pass, options.GitUserAuth is nil, this ends up a crash here.